crates/api_crud/src/tagline/update.rs

```diff
@@ -0,0 +1,33 @@
+use activitypub_federation::config::Data;
+use actix_web::web::Json;
+use lemmy_api_common::{
+  context::LemmyContext,
+  tagline::{TaglineResponse, UpdateTagline},
+  utils::is_admin,
+};
+use lemmy_db_schema::{
+  source::tagline::{Tagline, TaglineUpdateForm},
+  traits::Crud,
+  utils::naive_now,
+};
+use lemmy_db_views::structs::LocalUserView;
+use lemmy_utils::error::LemmyError;
+
+#[tracing::instrument(skip(context))]
+pub async fn update_tagline(
+  data: Json<UpdateTagline>,
+  context: Data<LemmyContext>,
+  local_user_view: LocalUserView,
+) -> Result<Json<TaglineResponse>, LemmyError> {
+  // Make sure user is an admin
+  is_admin(&local_user_view)?;
+
+  let tagline_form = TaglineUpdateForm {
+    content: Some(data.content.to_string()),
```

Comment:
One other thing I realized I missed, both for tagline update and create. `content` is a markdown field, so it needs to run some extra functions. Check out `crates/api_crud/src/comment/update.rs` , the `content` field, but you need:

`process_markdown_opt(&data.body, &slur_regex, &url_blocklist, &context).await?;`

and 

`is_valid_body_field`

Reply:
Thanks, would've totally missed that. Looks like there wasn't any validation in the previous solution, so good that it's being added now.